middleware/jwt.go

```diff
@@ -64,7 +64,12 @@ type (
 		// Optional. Default value "Bearer".
 		AuthScheme string
 
-		keyFunc jwt.Keyfunc
+		// KeyFunc defines a function to supply the key for a token verification.
+		// When a user-defined KeyFunc is provided, SigningKey, SigningKeys, and SigningMethod are ignored.
+		// Required. This or SigningKey or SigningKeys.
+		// Default value is nil, which uses a default internal implementation.
```

Comment:
It should not be nil actually so we should not document it that way.
Better change to something like
// Defaults to an internal implementation verifying the signing algorithm and selecting the proper key